src/app/views/PersonDetail.svelte

```diff
@@ -197,6 +197,18 @@
       <div class="-ml-16 flex flex-grow flex-col gap-4 xs:ml-0">
         <PersonAbout class="font-thin opacity-75" {pubkey} />
       </div>
+      <div class="-ml-16 flex flex-grow flex-col gap-4 xs:ml-0">
+        <PersonAbout class="font-thin opacity-75" {pubkey} />
+      </div>
+      {#if $profile?.website}
+        <Anchor
+          external
+          class="flex items-center gap-2 overflow-hidden overflow-ellipsis whitespace-nowrap"
+          href={ensureProto($profile.website)}>
+          <i class="fa fa-link text-accent" />
+          {stripProtocol($profile.website)}
+        </Anchor>
+      {/if}
```

Comment:
This isn't right, I moved website elsewhere, and personabout is now duplicated.

Reply:
I went through your commits, if I'm not mistaking, website is now inline at the tail end of PersonHandle and zapDisplay.

It will go hidden if PersonHandle and zapDisplay are too large, it is hidden on my profile for example.

<img width="641" alt="Screenshot 2024-10-25 at 11 15 02" src="https://github.com/user-attachments/assets/ce3e4ee2-5b88-49b8-91bf-58d763313dea">

The row is 80% of the container width atm, I can make it full size and add the flex-wrap condition on the container.
It will push the website to the next line.

<img width="637" alt="Screenshot 2024-10-25 at 11 22 54" src="https://github.com/user-attachments/assets/2421ba67-16a3-4fa2-bdee-8afef394b162">

Let me know your preference and I'll push the commit eventually